parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala

```diff
@@ -11,6 +11,7 @@ import scala.util.matching.Regex
   */
 object Tokens extends CrfTokenable {
 
```

Comment:
I have made a similar change to this class in my branch and have included a paoEndSuffix field - is it needed?